c2/c2-client-bundle/c2-client-service/src/main/java/org/apache/nifi/c2/client/service/operation/TransferDebugOperationHandler.java

```diff
@@ -178,6 +178,7 @@ private Optional<byte[]> createDebugBundle(List<Path> filePaths) {
         try (GzipCompressorOutputStream gzipCompressorOutputStream = new GzipCompressorOutputStream(byteOutputStream);
              TarArchiveOutputStream tarOutputStream = new TarArchiveOutputStream(gzipCompressorOutputStream)) {
             for (Path filePath : filePaths) {
+                tarOutputStream.setBigNumberMode(1);
```

Comment:
Can you change this to `TarArchiveOutputStream.BIGNUMBER_POSIX`? The value of `1` is equivalent to `BIGNUMBER_STAR`, which does not appear to be as widely supported according to the [commons-compress documentation](https://commons.apache.org/proper/commons-compress/tar.html).